### ISSUE TITLE
Automatic gas calculation doesn't work

### ISSUE THREAD

**Issue by russdirks** [MyCryptoHQ/MyCrypto#2199]
### Description of the Issue

Under some situations, mycrypto does not correctly calculate gas costs when sending ERC20 tokens:

### Steps to Reproduce

open keystore wallet & unlock
paste destination address
select an erc20 token from the drop down
type in the token amount to send
click advanced
click send transaction & confirm
error message : "transaction gas is too low ..."

### Description of Your Machine

Windows 10 PC
Standalone mycrypto app

**Comment by tayvano** [comment]
Were you sending transactions back-to-back? That error message is unclear but I think relates to attempting to replace a transaction already in the queue. 

`Transaction gas price is too low. There is another transaction with same nonce in the queue. Try increasing the gas price or incrementing the nonce.`

**Comment by tayvano** [comment]
This should be resolved as off the latest release (1.5.3). Please comment & re-open if you are still experiencing the issue. Thanks for the report.

**Comment by russdirks** [comment]
The issue is still present.  Maybe if I use pictures the issue will become clearer.  Here is what the screen looks like right after clicking the Advanced button (see the repro steps above):

![untitled1](https://user-images.githubusercontent.com/7891737/50551660-5168c280-0c39-11e9-9efe-6d0fb8708ee4.png)

Notice that I have selected an ERC20 token from the dropdown, the "Automatically calculate gas limit" is pre-selected for me, **and the gas limit is 2100**.  That would be correct if it was an ETH transfer, but it is too low for an ERC20 transfer.

I found a work-around, which probably will also highlight what the problems is.  If I **uncheck**, and then **re-check** the "Automatically calculation gas limit", the gas limit changes to **52094**, which is correct:

![untitled2](https://user-images.githubusercontent.com/7891737/50551693-cd630a80-0c39-11e9-8fa1-78fe487f499e.png)
